core/src/integration-test/java/com/scalar/db/storage/StorageMultipleClusteringKeyScanIntegrationTestBase.java

```diff
@@ -63,13 +70,17 @@
 
   private static long seed;
 
+  private static ExecutorService executorService;
```

Comment:
I also made this multiple clustering key scan integration test run concurrently with multi threads.